### PR TITLE
Add console to docs for inner hits, explain, and friends

### PR DESCRIPTION
**Pull request by MaineC**
This adds CONSOLE to inner hits, explain, field-stats, request body, scroll docs.

Relates to #18160 

@nik9000 Care to take a look?

### CODE REVIEW

File: docs/reference/search/field-stats.asciidoc [hunk 1]
```diff
@@ -9,22 +9,46 @@ available in the Lucene index. This can be useful to explore a dataset which
 you don't know much about. For example, this allows creating a histogram
 aggregation with meaningful intervals based on the min/max range of values.
 
+For the following examples, lets assume the following indexed data:
```
Comment (nik9000) [inline docs/reference/search/field-stats.asciidoc]: s/lets/let's/

File: docs/reference/search/request/scroll.asciidoc [hunk 2]
```diff
@@ -38,29 +38,29 @@ should keep the ``search context'' alive (see <<scroll-search-context>>), eg `?s
...
 [source,js]
 --------------------------------------------------
-curl -XGET <1> 'localhost:9200/_search/scroll' <2> -d'
+GET <1> /_search/scroll <2> 
 {
     "scroll" : "1m", <3>
     "scroll_id" : "c2Nhbjs2OzM0NDg1ODpzRlBLc0FXNlNyNm5JWUc1" <4>
 }
-'
 --------------------------------------------------
```
Comment (nik9000) [inline docs/reference/search/request/scroll.asciidoc]: `// CONSOLE`? You can do a `// TEST[s/c2Nhbjs2OzM0NDg1ODpzRlBLc0FXNlNyNm5JWUc1/"$body.scroll_id"/]` or something like that?

**Comment by nik9000** [comment]
Left two suggestions but LGTM.

File: docs/reference/search/request/scroll.asciidoc (hunk 2 above)
Comment (MaineC) [inline docs/reference/search/request/scroll.asciidoc]: How would I get a scroll id from a previous result?

Comment (nik9000) [inline docs/reference/search/request/scroll.asciidoc]: It ought to be available at `$body._scroll_id` so you can pick it up with `s//`.

**Comment by nik9000** [comment]
@MaineC could you finish this one up? I might start making more of these sorts of PRs if you are willing to review them.

**Comment by MaineC** [comment]
@nik9000 I updated the PR; will merge if CI is green (only needed a few IMHO minor changes from what you already LGTM'ed earlier to make things work again).

TODO left for later: In explain.asciidoc I had to add a "\n" to the expected response as this is what the response seems to be looking like. My searching skills left me trying to figure out where in our code this is coming from.

File: docs/reference/search/request/scroll.asciidoc (hunk 2 above)
Comment (MaineC) [inline docs/reference/search/request/scroll.asciidoc]: I've left this open while finishing the PR up, I couldn't figure out how to carry the scroll_id from a response of some request over to the next request. Would be great if you could look at the doc for me and notify me when you fix that so I can read and learn.

Comment (nik9000) [inline docs/reference/search/request/scroll.asciidoc]: `// TEST[continued]` ought to do that. It is in `docs/README.asciidoc:19`.

**Comment by MaineC** [comment]
jenkins, test it

**Pull request merged**